<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealTimeDataWidget</class>
 <widget class="QWidget" name="RealTimeDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
/*背景色*/
	background-color: rgb(210, 220, 255);
/*边框样式*/
border-style: outset;
/*边框宽度为4像素*/
border-width: 4px;
/*边框圆角半径*/
border-radius: 10px;
/*边框颜色*/
border-color: rgba(255, 225, 255, 30);
/*字体*/
font: bold 20px;
/*字体颜色*/
color: rgb(35, 35, 35);
/*填衬*/
padding: 6px;
}

QPushButton:disabled{
background-color: rgba(211, 211, 211, 180);
color: rgba(39, 39, 39, 100);
}

/*鼠标悬停在按钮上时*/
QPushButton:hover{
	background-color: rgb(102, 102, 255,200);
border-color: rgba(255, 225, 255, 200);
color:rgba(0, 0, 0, 200);
}

/*按钮被按下时*/
QPushButton:pressed {
background-color: rgba(102, 102, 255, 200);
border-color: rgba(255, 225, 255, 30);
border-style: inset;
color:rgba(0, 0, 0, 100);
}

/*********标签*************/
QLabel{
	font: bold 17px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,1,1,1,1,1,3,1">
     <item>
      <widget class="QLabel" name="label_gid">
       <property name="text">
        <string>设备分组ID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>coBox_gid</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="coBox_gid">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string>设备编号</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>lineE_id</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineE_id">
       <property name="text">
        <string>001</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_interval">
       <property name="text">
        <string>查询间隔(ms)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>lineE_interval</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineE_interval">
       <property name="text">
        <string>5000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_type">
       <property name="text">
        <string>当前设备型号：</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pBtn_startQuery">
         <property name="text">
          <string>开始查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pBtn_stopQuery">
         <property name="text">
          <string>暂停查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tWidget_realtimeData">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>11</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>12</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>13</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>14</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>15</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>16</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>17</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>18</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>20</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>经度</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>纬度</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>液压系统油量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>液压系统油温</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>支腿回路压力</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>支腿回路流量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>支腿回路振动</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>起升回路压力</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>起升回路流量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>起升回路油温</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>柴油机压力</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>柴油机油量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>柴油机振动</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
